doc/Command-Reference.md

```diff
@@ -1371,6 +1371,33 @@ When the optional argument "max_priority"  is specified, each rule’s priority
 
 Go Back To [Beginning of the document](#) or [Beginning of this section](#acl)
 
+**config acl add table**
+
+This command is used to create new ACL tables.
+
+- Usage:
+  ```
+  config acl add table [OPTIONS] <table_name> <table_type> [-d <description>] [-p <ports>] [-s (ingress | egress)] [--expand_vlans]
+  ```
+
+- Parameters:
+  - table_name: The name of the ACL table to create.
+  - table_type: The type of ACL table to create (e.g. "L3", "L3V6", "MIRROR")
+  - description: A description of the table for the user. (default is the table_name)
+  - ports: A comma-separated list of ports to add to the table. (default is all front-panel ports)
```

Comment:
>A comma-separated list of ports to add to the table. (default is all front-panel ports) [](start = 11, length = 87)

Since it support ports, interfaces (single port, portchannel or vlan). Let's make the usage/document clear.
Not easy to describe it in the `[-p <ports>]`, it's okay to keep it as-is.

Reply:
Let me know if the updated description is clear enough